src/test/java/me/dinowernli/grpc/polyglot/io/MessageReaderTest.java

```diff
@@ -56,6 +54,13 @@ public void handlesEmptyStream() {
   }
 
   @Test
+  public void handlePrimitives() {
+    reader = MessageReader.forFile(dataFilePath("request_with_primitives.pb.ascii"), DESCRIPTOR);
+    ImmutableList<DynamicMessage> result = reader.read();
+    assertThat(result).isNotEmpty();
```

Comment:
Slightly odd that you don't assert that the number field has the correct value here.